onnxruntime/test/python/transformers/test_flash_attn_cuda.py

```diff
@@ -1155,25 +1153,7 @@ def parity_check_mha(
         out_ref, _ = attention_ref(q, k, v, None, None, 0.0, None, causal=False)
         out_ref = out_ref.detach().cpu().numpy()
 
-    # Compare results
-    all_close = numpy.allclose(out, out_ref, rtol=rtol, atol=atol, equal_nan=True)
-    correct = GREEN + "True" + RESET if all_close else RED + "False" + RESET
-    print(
-        " B:",
-        config.batch_size,
-        " S:",
-        config.sequence_length,
-        " N:",
-        config.num_heads,
-        " kvN:",
-        config.kv_num_heads,
-        " h:",
-        config.head_size,
-        " Mean Error:",
-        numpy.mean(numpy.abs(out - out_ref)),
-        correct,
-    )
-    return all_close
+    numpy.testing.assert_allclose(out, out_ref, rtol=rtol, atol=atol, equal_nan=True)
```

Comment:
please use err_msg= parameter to show the config. Otherwise, we do not know which config is not passed.

Reply:
Added, force trigged as

```
E           AssertionError: 
E           Not equal to tolerance rtol=2e-08, atol=2e-08
E            with PromptConfig(batch_size=3, q_sequence_length=127, kv_sequence_length=127, buffer_sequence_length=262, num_heads=32, kv_num_heads=8, head_size=16, ep=rocm)
E           Mismatched elements: 11184 / 100608 (11.1%)
E           Max absolute difference: 0.001953
E           Max relative difference: 51.
E            x: array([[[[-0.655   , -1.375   ,  0.755   , ...,  0.9043  ,  0.893   ,
E                      1.108   ],
E                    [ 1.102   , -0.5854  , -0.5405  , ..., -1.672   ,  0.1144  ,...
E            y: array([[[[-0.6553  , -1.375   ,  0.755   , ...,  0.9043  ,  0.893   ,
E                      1.107   ],
E                    [ 1.101   , -0.5854  , -0.5405  , ..., -1.672   ,  0.1144  ,...
```